docs/faq/component-libraries.md

```diff
@@ -0,0 +1,47 @@
+
+  A family of web components built with first class participation in the Twig templating system for PHP in mind backed by an expansive catalog of usage variants.
+- [Elix](https://component.kitchen/elix)
+
+  The Elix project aims to create a universal library of all general-purpose user interface patterns commonly found in desktop and mobile UIs, where each pattern is implemented as a well-designed, attractive, high-quality, performant, accessible, localizable, and extensively customizable web component.
+- [Helix UI](https://rackerlabs.github.io/helix-ui/)
+
+  The HelixUI library provides front-end developers a set of reusable CSS classes and HTML Custom Elements that adhere to Helix design standards as outlined by Rackspace.
+- [Ionic UI Components](https://ionicframework.com/docs/components)
+
+  Take advantage of the component system that powers Ionic applications with a large ecosystem to choose from and indepth usage instructions no matter the framework with which your project is built.
+- [Lightning Web Components by SalesForce](https://developer.salesforce.com/docs/component-library/overview/components)
+
+  Lightning Web Components are not the go to standard for developing applications in the SalesForce ecosystem, and with the power of web components they can be the basis of applications outside of their ecosystem, too! 
+- [Material Web Components](https://material-components.github.io/material-components-web-components/demos/index.html)
+
+  Material Design Component from the team behind the Material Design specification. Stay as close to the changing specification as possible with these web components from Google's own Material Design team.
+- [Morningstar](http://designsystem.morningstar.com/components/component-status.html)
+
+  The Morningstar Design System combines vanilla HTML/CSS with web components in just the right proportions to empower the design and development of wide reaching content and functionality.
+- [SAP UI5](https://sap.github.io/ui5-webcomponents/)
+
+  Get the power and flexibility of SAP'ss UI5 rendering stack with the ergonomics and ease of use of Web Components, and bring enterprise-grade features, Fiori UX and themeability home to your application.
+- [Smart HTML Elements](https://www.htmlelements.com)
+
+  With the goal of helping to deliver modern responsive websites and applications that work on any device and are pleasure to use, the Smart HTML Elements collection is one the most feature rich and comprehensive javascript user interface frameworks.
+- [Vaadin](https://vaadin.com/components)
+
+  Vaadin has a comprehensive set of beautifully crafted, performant, and adaptable UI components for modern mobile-first Web apps. They are the ideal building blocks for Progressive Web Applications.
+- [Wired Elements](https://wiredjs.com/)
+
+  A set of common UI elements with a hand-drawn, sketchy look. These can be used for wireframes, mockups, or just the fun hand-drawn look.
```

Comment:
```suggestion
  A set of common UI elements with a hand-drawn, sketchy look. These can be used for wireframes, mockups, or just the fun art style.
```

Reply:
I copied this direct from their site, so I'm gonna let it slide.